src/inc/msquic_winuser.h

```diff
@@ -26,8 +26,16 @@
 #include <windows.h>
 #include <ws2def.h>
 #include <ws2ipdef.h>
+
```

Comment:
Personally, I'd prefer not to have all the extra spaces around these #pragma changes.

Reply:
Done